app/templates/main/search-transferring-body.html

```diff
@@ -91,20 +90,20 @@ <h2 class="govuk-body browse__body">You are viewing</h2>
                                                 {% for record in results %}
                                                     {% set main_loop_index = loop.index %}
                                                     <tr class="govuk-table__row govuk-table__row--primary">
-                                                        {% set first_result_key = record['highlight'].keys() | list | first | replace('_', ' ') | capitalize %}
-                                                        {% set first_result_value = record['highlight'].values() | list | first | join(' ... ')| clean_tags | safe %}
-                                                        {% set field_count = record['highlight'].keys() | length - 1 %}
+                                                        {% set first_field = record['highlight'].keys() | list | first | replace('_', ' ') | capitalize %}
+                                                        {% set first_result_value = record['highlight'].values() | list | first | join(' ... ')| clean_tags(highlight_tag) | safe %}
+                                                        {% set total_fields = record['highlight'].keys() | length %}
                                                         <td class="govuk-table__cell">
                                                             <div class="govuk-table__cell__data">
                                                                 <details class="details-toggle-{{ main_loop_index }}"
                                                                          {% if open_all %}open{% endif %}>
                                                                     <summary />
                                                                 </details>
                                                                 {% if record['highlight'] %}
-                                                                    {% if field_count > 0 %}
-                                                                        {{ first_result_key }} +{{ field_count }}
+                                                                    {% if total_fields > 1 %}
+                                                                        {{ first_field }} +{{ total_fields }}
```

Comment:
regression of the + 1 bug: this needs to be `total_fields - 1` now

let's add a test for it